zeppelin-user-impersonation[3016].md

```diff
@@ -0,0 +1,152 @@
+# zeppelin-user-impersonation
+How to setup Zeppelin User Impersonation with 0.7.x
+Zeppelin User Impersonation Per User and Isolated Per User
+This document completely focuses how to set up Zeppelin Interpreter Per User, Isolated Per User with User Impersonation using zeppelin 0.7.0/0.7.2/0.7.3 or HDP 2.6
+
+How to set up user impersonation at Zeppelin Shell Interpreter level with shiro authentication can found from [here](https://zeppelin.apache.org/docs/0.7.0/manual/userimpersonation.html)
+What the different kinds of Interpreter binding modes is well discussed in the upcoming version 0.8.0 of Zeppelin [documentation](https://zeppelin.apache.org/docs/0.8.0-SNAPSHOT/usage/interpreter/interpreter_binding_mode.html)
+
+Before setting up the Zeppelin User Impersonation make sure you have integrated your zeppelin and your cluster with AD/LDAP, below is the shiro.ini for AD authentication: </br>
+<i>
```

Comment:
shiro setting could be referenced by the `docs/setup/security/shiro_authentication.md`, we don't repeat it again here.